app/index.html

```diff
@@ -39,21 +39,26 @@
               <span class="icon-bar"></span>
             </button>
 
-            <a class="navbar-brand" href="#/">Note List</a>
+           
           </div>
 
           <div class="collapse navbar-collapse" id="js-navbar-collapse">
 
+          <div class= "menu"> <!--added another class so that the navigation links are in a box-->
```

Comment:
By creating a new menu, you don't inherit the properties that make the menu mobile friendly and work with the styling of bootstrap.